models/_sector/nft/trades/ethereum/platforms/_schema.yml

```diff
@@ -0,0 +1,92 @@
+version: 2
+
+models:
+ - name: archipelago_ethereum_base_trades
+   meta:
+     blockchain: ethereum
+     sector: nft
+     project: archipelago
+     contributors: 0xRob
+   config:
+     tags: ['ethereum', 'nft', 'trades', 'archipelago']
+   description: "Archipelago base trades"
+   tests:
+     - dbt_utils.unique_combination_of_columns:
+         combination_of_columns: ['block_number','tx_hash','sub_tx_trade_id']
+     - dbt_utils.equal_rowcount:
```

Comment:
this is great, specifically comparing to source

Reply:
but i notice the others can't necessarily apply the same test. will that always be the case due to complexity or will they eventually add similar tests?